guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Web-UI-with-a-Chrome-browser.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Web_UI-with-a-Chrome-browser_{context}"]
```

Comment:
```suggestion
[id="Using_IdM_credentials_to_log_in_to_the_{ProjectWebUI}-with-a-Chrome-browser_{context}"]
```

Please either use `-` or `_` in your anchor. All file names must you dashes after the first underscore after the source file type (like `proc_`).